tracee-ebpf/tracee/events_processor.go

```diff
@@ -255,6 +261,39 @@ func (t *Tracee) processEvent(event *external.Event) error {
 			return fmt.Errorf("error parsing cgroup_rmdir args: %v", err)
 		}
 		t.containers.CgroupRemove(cgroupId)
+
+	case DetectHookedSyscallsEventID:
+		doOnce.Do(func() {
+			hookedSyscallNumberMap, err := t.bpfModule.GetMap("hooked_syscalls_map")
+			if err == nil {
+				hookedSyscallData := make([]kernelsyms.HookedSyscallData, 0, 0)
+				i := 0
+				for true {
```

Comment:
Instead of looping with unknown bounds, you can pass the number of elements as an argument of the event